src/MySqlConnector/Core/SingleCommandPayloadCreator.cs

```diff
@@ -246,23 +246,52 @@ private static bool WriteCommand(IMySqlCommand command, ByteBufferWriter writer,
 	{
 		var isSchemaOnly = (command.CommandBehavior & CommandBehavior.SchemaOnly) != 0;
 		var isSingleRow = (command.CommandBehavior & CommandBehavior.SingleRow) != 0;
-		if (isSchemaOnly)
+		if (isSchemaOnly || isSingleRow)
 		{
-			ReadOnlySpan<byte> setSqlSelectLimit0 = "SET sql_select_limit=0;\n"u8;
-			writer.Write(setSqlSelectLimit0);
-		}
-		else if (isSingleRow)
-		{
-			ReadOnlySpan<byte> setSqlSelectLimit1 = "SET sql_select_limit=1;\n"u8;
-			writer.Write(setSqlSelectLimit1);
```

Comment:
I probably should have inlined these constants in https://github.com/mysql-net/MySqlConnector/commit/f8a0a4013a0389b9ac3bd44068f3cbdb237c1aba#diff-15fac83b96ddd1a37bdce12382e8398f3e7cecd40ff35e6ad6a8994c5b09ab33; there's no reason to have a temporary variable.